### ISSUE TITLE
Dashboard UI Enhancement

### ISSUE THREAD

**Issue by nisha-laur** [CircuitVerse/CircuitVerse#4424]
### Is your feature request related to a problem? Please describe

The current UI of the dashboard lacks organization, and the screen space is not effectively utilized. This issue aims to address these concerns by implementing an approved design, as detailed in the [Figma file](https://www.figma.com/file/LqJ3ezdv1dxaYe74ejQOhP/circuitverse-ui-updates?type=design&node-id=307%3A32&mode=design&t=7gQExApKpX4Yov22-1)

### Describe the solution you'd like

I propose creating a pull request to implement the approved design from the Figma file. This design has been carefully crafted to improve the organization of the dashboard and make optimal use of the available screen space. By merging this pull request, we can enhance the overall user experience and provide a more visually appealing and user-friendly dashboard.
Before :
![Group 32](https://github.com/CircuitVerse/CircuitVerse/assets/104618576/04fd3479-09d5-44eb-8d05-0655007caea4)
After:
![Frame 1](https://github.com/CircuitVerse/CircuitVerse/assets/104618576/6c5d2536-c48a-4ec5-8b1c-459d84f9482f)

### Are you working on this?

Yes

**Comment by tanmoysrt** [comment]
Hi @nisha-laur , it's looks great !

If you have re-designed using figma, put a figma link on description
And post that in slack `#ux` as well.

**Comment by nisha-laur** [comment]
Hi @tanmoysrt  , I have added the link under the Figma link (https://www.figma.com/file/LqJ3ezdv1dxaYe74ejQOhP/circuitverse-ui-updates?type=design&node-id=307%3A32&mode=design&t=lH1VSLd14WryEAoY-1) name .

**Comment by tanmoysrt** [comment]
> Hi @tanmoysrt , I have added the link under the Figma link (https://www.figma.com/file/LqJ3ezdv1dxaYe74ejQOhP/circuitverse-ui-updates?type=design&node-id=307%3A32&mode=design&t=lH1VSLd14WryEAoY-1) name .

Thanks

**Comment by Asrani-Aman** [comment]
Hello @tanmoysrt , can i make a pull request of merging the code of this proposed Figma design

**Comment by tanmoysrt** [comment]
> Hello @tanmoysrt , can i make a pull request of merging the code of this proposed Figma design  

Sorry, but @nisha-laur is interested in doing that

**Comment by Asrani-Aman** [comment]
yes sure sir

**Comment by nisha-laur** [comment]
@Asrani-Aman yeah Actually I am waiting for the approval of this design and once it gets approved , Will start working on it asap.
